roxie/ccd/ccdmain.cpp

```diff
@@ -1116,11 +1116,13 @@ int CCD_API roxie_main(int argc, const char *argv[], const char * defaultYaml)
             mtu_size = 1400;    // upper limit on outbound buffer size - allow some header room too
             roxiemem::setDataAlignmentSize(0x400);
         }
-        unsigned pinterval = topology->getPropInt("@systemMonitorInterval",1000*60);
+
```

Comment:
@richardkchapman does it make sense to keep the perfmonitor around to be exposed via control mess?

Reply:
I doubt it. But the comment here suggests that there is code in ccdquery.cpp you should be changing too.